docs/admin/writing-components.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Writing custom components
+---
+
+React components must be wrapped into `Component` higher order component to support [data binding](/admin/data-binding/overview.md). More complex components with logic, hooks etc. must use second argument of this function with a "static render" implementation.
```

Comment:
Add a note, that components if component doesn't directly use databinding it's children are automatically analyzed (such as Box), but when it implements static render, it has to pass children as well (such as Repeater)...